sdks/python/apache_beam/runners/worker/sdk_worker_test.py

```diff
@@ -115,8 +115,9 @@ def _check_fn_registration_multi_request(self, *args):
           {item.id: item
            for item in process_bundle_descriptors})
 
-  def test_fn_registration(self):
-    self._check_fn_registration_multi_request((1, 4), (4, 4))
+
+#   def test_fn_registration(self):
+#     self._check_fn_registration_multi_request((1, 4), (4, 4))
```

Comment:
if we can, we should test the registration lookup failure case.

The success case will be easily be covered by other tests.